com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageHandler.cs

```diff
@@ -190,7 +190,12 @@ public void HandleDestroyObject(ulong clientId, Stream stream)
             using (var reader = PooledNetworkReader.Get(stream))
             {
                 ulong networkId = reader.ReadUInt64Packed();
-                NetworkManager.SpawnManager.OnDespawnObject(networkId, true);
+                if (!NetworkManager.SpawnManager.SpawnedObjects.TryGetValue(networkId, out NetworkObject networkObject))
```

Comment:
I think here we did get a "Trying to destroy object {networkObject.NetworkObjectId} but it doesn't seem to exist anymore!" before and we are not getting that anymore. Maybe we should keep that log.

Reply:
Yes that's a good point - I can just pass in network object without the if statement as there's also a null check in OnDespawnObject now (I'll also swap the order of the null check and the SpawnedObjects check to make sure we get the log message first).